include/z64speed_meter.h

```diff
@@ -0,0 +1,18 @@
+#ifndef Z64SPEED_METER_H
+#define Z64SPEED_METER_H
+
+#include "PR/ultratypes.h"
+
+typedef struct SpeedMeter {
+    /* 0x00 */ char unk_00[0x1C];
+} SpeedMeter; // size = 0x20
+
+struct GameState;
+struct GraphicsContext;
```

Comment:
iirc we have been putting those struct forward declaration at the top of the file, before other struct definitions. Could you move them up?